### PR TITLE
Complete the home work # 1

### PR DESCRIPTION
**Pull request by AnastasiiaNashchokina**

### CODE REVIEW

File: src/main/java/App.java [hunk 1]
```diff
@@ -1,5 +1,7 @@
 public class App {
     public static void main(String[] args) {
+
```
Comment (AntBelyi) [inline src/main/java/App.java:3]: Extra line. remove it please

File: src/main/java/App.java [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
         System.out.println("Hello, my name is Java app for ALevel students!");
     }
+
```
Comment (AntBelyi) [inline src/main/java/App.java:6]: Extra line. remove it please

File: src/main/java/HW_1/Task1.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package HW_1;
+
+public class Task1 {
+    public static void main(String[] args) {
+
```
Comment (AntBelyi) [inline src/main/java/HW_1/Task1.java:5]: Extra line. remove it please

File: src/main/java/HW_1/Task1.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+        String name = "Anastasiia";
+        int age = 30;
+        float weight = 53.5f;
+
+        System.out.println("name=" + name +" " + "age=" + age +" " + "weight=" + weight);}
```
Comment (AntBelyi) [inline src/main/java/HW_1/Task1.java:10]: "}" symbol should be on the next line

File: src/main/java/HW_1/Task2.java [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+public class Task2 {
+    public static void main(String[] args) {
+        int a = 1;
+        int b = 2;
+        System.out.println("a="+ a + " "  + "b=" + b);
+        int temp = a;
+        a = b;
+        b = temp;
+        System.out.println("a="+ a +" " + "b=" + b);}
```
Comment (AntBelyi) [inline src/main/java/HW_1/Task2.java:11]: "}" symbol should be on the next line

File: src/main/java/HW_1/Task2.java [hunk 6]
```diff
@@ -0,0 +1,12 @@
+package HW_1;
+
+public class Task2 {
+    public static void main(String[] args) {
+        int a = 1;
+        int b = 2;
+        System.out.println("a="+ a + " "  + "b=" + b);
+        int temp = a;
```
Comment (AntBelyi) [inline src/main/java/HW_1/Task2.java:8]: lets rename it as helper

File: src/main/java/HW_1/Task2.java (hunk 5 above)
Comment (AntBelyi) [inline src/main/java/HW_1/Task2.java:11]: add an extra space after "+"

File: src/main/java/HW_1/Test3.java [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+    public static void main(String[] args) {
+        int a = 5;
+        int b = 3;
+        a = a + b;
+        b = a - b;
+        a = a - b;
+        a = a + b - (b = a);
+        a = a ^ b;
+        b = b ^ a;
+        a = a ^ b;
```
Comment (AntBelyi) [inline src/main/java/HW_1/Test3.java:13]: Do we really need all the line?
Screenshot_5

File: src/main/java/HW_1/Test3.java [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+        System.out.println("a="+ a + " "  + "b=" + b);
```
Comment (AntBelyi) [inline src/main/java/HW_1/Test3.java:14]: Add an extra space after "a=" before "+"

File: src/main/java/App.java (hunk 1 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/App.java:3]: fixed

File: src/main/java/App.java (hunk 2 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/App.java:6]: fixed

File: src/main/java/HW_1/Task1.java (hunk 3 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/HW_1/Task1.java:5]: fixed

File: src/main/java/HW_1/Task1.java (hunk 4 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/HW_1/Task1.java:10]: fixed

File: src/main/java/HW_1/Task2.java (hunk 5 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/HW_1/Task2.java:11]: fixed

File: src/main/java/HW_1/Task2.java (hunk 6 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/HW_1/Task2.java:8]: fixed

File: src/main/java/HW_1/Task2.java (hunk 5 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/HW_1/Task2.java:11]: fixed

File: src/main/java/HW_1/Test3.java (hunk 7 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/HW_1/Test3.java:13]: fixed

File: src/main/java/HW_1/Task1.java (hunk 4 above)
Comment (AntBelyi) [reply src/main/java/HW_1/Task1.java:10]: not fixed 
Screenshot_16

**Pull request closed without merging**